package.json

```diff
@@ -22,7 +22,7 @@
     "ci": "yarn test && yarn test:e2e"
   },
   "dependencies": {
-    "@date-io/moment": "^1.3.13",
+    "@date-io/date-fns": "^2.10.6",
```

Comment:
According to the material-ui pickers [docs](https://material-ui-pickers.dev/getting-started/installation#peer-library), `@date-io/date-fns` should be **v1**